src/snapred/backend/dao/ingredients/CalibrationMetricsWorkspaceIngredients.py

```diff
@@ -16,4 +16,4 @@ class CalibrationMetricsWorkspaceIngredients(BaseModel):
     """
 
     calibrationRecord: CalibrationRecord
-    timestamp: Optional[int] = None
+    timestamp: Optional[float] = None
```

Comment:
This surprised me as well.  I think what they are doing is working to prevent _ambiguity_ between systems.  That is a `float` timestamp in seconds is the least confusing way to represent it.